src/ontology/mondo-edit.obo

```diff
@@ -327573,8 +327573,7 @@ property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/
 [Term]
 id: MONDO:0014855
 name: intellectual disability, autosomal dominant 42
-def: "Any autosomal dominant non-syndromic intellectual disability in which the cause of the disease is a mutation in the GNB1 gene." [MONDO:patterns/disease_series_by_gene]
-comment: Editor note: check status as syndromic
+def: "Any autosomal dominant non-syndromic intellectual disability in which the cause of the disease is a mutation in the GNB1 gene. It is characterized by global developmental delay, intellectual disability, hypotonia, structural brain abnormalities, and seizures. Other less common findings include dystonia, visual impairment, behavior problems, growth delay, craniofacial defects, and genitourinary abnormalities in males." [https://clinicalgenome.org/affiliation/40006/, MONDO:patterns/disease_series_by_gene]
```

Comment:
@nicolevasilevsky I noticed that the submitter and OMIM mention that this is caused by a "heterozygous mutation in the GNB1 gene" https://www.omim.org/entry/616973?search=616973&highlight=616973

Do we add detailed information like that into the definition?

Reply:
yes, good catch, I'll add this